apps/src/code-studio/components/progress/StageLock.jsx

```diff
@@ -52,10 +52,12 @@ const StageLock = React.createClass({
     }
     return (
       <div style={{display: 'inline-block'}}>
+        /* className used only for purposes of uitest */
         <button
           style={progressStyles.blueButton}
           onClick={this.openLockDialog}
           disabled={this.props.saving}
+          className="uitest-locksettings"
```

Comment:
My goal in doing this was mostly to make it clear that you don't need to go search through CSS files to see how this component actually renders. I think I've recommended doing this to others before as well.